.github/workflows/ci-daily-build.yml

```diff
@@ -0,0 +1,42 @@
+name: ci-ifcopenshell-daily
+
+on:
+  push:
+    paths:
+      - src/**
+      - conda/**
+      - .github/workflows/ci-daily-build.yml
+    branches:
+      - pr-daily-builds
+jobs:
+  test:
+    name: ${{ matrix.platform.name }}-${{ matrix.pyver.name }}
+    runs-on: ${{ matrix.platform.distver }}
+    defaults:
+      run:
+        shell: bash -l {0}
+    strategy:
+      fail-fast: false
+      matrix:
+        pyver: [
+          { name: py39, distver: '3.9' },
+          { name: py310, distver: '3.10'}
+        ]
+        platform: [
+          { name: Windows, distver: windows-latest, short: 'win-64' },
+          { name: Linux, distver: ubuntu-latest, short: 'linux-64' },
+          { name: macOS, distver: macos-latest, short: 'osx-64' }
```

Comment:
Should we use slightly older versions of OSes so that there is a greater chance that the binaries can be ran on older systems. E.g on linux this mostly depends on the version of glibc.

Reply:
Sure, that's fine by me. We could in principle test on multiple versions for each platform and just add an extra flag on the specific version of each platform we wish to upload to conda. Like so:

```
platform: [
          { name: Windows, distver: windows-latest, short: 'win-64', upload: false },
          { name: Windows, distver: windows-2019, short: 'win-64', upload: true },
          ...
]
```

That way we can test on all versions and still only compile the uploaded conda-packages for a specific platform version. If that makes any sense?

Reply:
Great idea. Maybe it actually doens't matter so much. At least on Windows I think conda tries to find the appropriate compiler for the given python version. But I'm not sure how this works on nixes where there isn't a single authoritative python distribution and distro/compiler/glibc are more closely tied together.